project/Versions.scala

```diff
@@ -0,0 +1,33 @@
+import scala.util.Properties
+
+import sbt._
```

Comment:
?

Reply:
Resolved in https://github.com/azavea/franklin/pull/29/commits/0796dbe7808d9fa53006fc23c3e57a1660bef9a8